spec/rubocop/target_finder_spec.rb

```diff
@@ -428,17 +428,17 @@
       allow(config).to receive(:for_all_cops).and_return(exclude_property)
       allow(config_store).to receive(:for).and_return(config)
 
-      expect(found_basenames).not_to include('ruby1.rb')
-      expect(found_basenames).to include('ruby3.rb')
-      expect(found_basenames).to include('ruby4.rb')
+      expect(found_basenames.include?('ruby1.rb')).to be(false)
+      expect(found_basenames.include?('ruby3.rb')).to be(true)
+      expect(found_basenames.include?('ruby4.rb')).to be(true)
     end
 
     it 'works if patterns are empty' do
       allow(Dir).to receive(:glob).and_call_original
       allow_any_instance_of(described_class).to receive(:wanted_dir_patterns).and_return([])
 
       expect(Dir).to receive(:glob).with([File.join(base_dir, '**/*')], flags)
-      expect(found_basenames).to include(
+      expect(found_basenames).to include( # rubocop:todo RSpec/PredicateMatcher
```

Comment:
This is a false positive, perhaps nullification seems like a better way to go. WDYT?

Reply:
It certainly looks good. Note as another way of thinking, there was no rush to update the gem, it could have been done after the bug will be fixed :-)

Reply:
RuboCop RSpec 2.18 has been released. Can you update this PR to resolve this false positive?
https://github.com/rubocop/rubocop-rspec/releases/tag/v2.18.0

Reply:
I updated this PR.